addons/account/views/account_view.xml

```diff
@@ -814,6 +815,7 @@ action = model.setting_init_bank_account_action()
                     <field name="sequence"/>
                     <field name="note"/>
                     <field name="transaction_type"/>
+                    <field name='company_id' options="{'no_create': True}" groups="base.group_multi_company"/>
```

Comment:
It is weird that we can see the company but not the journal, as the company is related on the journal